chromium_src/chrome/browser/component_updater/trust_token_key_commitments_component_installer.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/component_updater/trust_token_key_commitments_component_installer.h"
+
+#define RegisterTrustTokenKeyCommitmentsComponentIfTrustTokensEnabled \
+  RegisterTrustTokenKeyCommitmentsComponentIfTrustTokensEnabled_ChromiumImpl
+
+#include "src/chrome/browser/component_updater/trust_token_key_commitments_component_installer.cc"
+#undef RegisterTrustTokenKeyCommitmentsComponentIfTrustTokensEnabled
+
+namespace component_updater {
+
+// We do not support TrustTokens aka PrivateStateTokens aka FeldgePst
+void RegisterTrustTokenKeyCommitmentsComponentIfTrustTokensEnabled(
```

Comment:
what's the `IfEnabled` part? What decides if it's enabled or not?

Reply:
The upstream function used to check 
```
-  if (!base::FeatureList::IsEnabled(network::features::kPrivateStateTokens) &&
-      !base::FeatureList::IsEnabled(network::features::kFledgePst)) {
-    return;
-  }
```

This was removed but it seems like they just didn't rename the function.